AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -213,16 +212,16 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
                 ant_pointing[name]["delta_azimuth_std"] =0.0#calc
                 for pol in pol_ind:
                     ant_pointing[name]["beam_height_%s"%(pol)]     = w_average(gaussian_height[pol_ind[pol],ant],axis=0,weights=1./gaussian_height_std[pol_ind[pol],ant]**2)
-                    ant_pointing[name]["beam_height_%s_std"%(pol)] = np.sqrt(np.nansum(1./gaussian_height_std[pol_ind[pol],ant]**2) )
```

Comment:
I think @spassmoor should comment here... This is actually `(1 / std ** 2) = (1 / var)`, i.e. a typical weight. Are these being used as weights anywhere?

Reply:
_the motivation for my change is as follows:_
1. as far as i can tell the primary downstream use for weights would use "delta_azimuth_std" & "delta_elevation_std" (and these are currently ==0 in this code). see for example fit_pointing_model.py, where the "delta_*_std" are used to normalize the residuals for the chi^2 metric. so in that case clearly "_std" mean "standard deviation" not "1/var".
2. besides for "delta_*_std", this code seems to be similar to analyse_point_source_scans.py#reduce_compscan_with_uncertainty(), which also generates results with "_std" labels and for which the values are indeed "standard deviation".

Reply:
@spassmoor, please comment!

Reply:
Yes this appears to be a typo where I have put a weight instead of a standard deviation. The code was originally written to produce output "Identical" to the analyse_point_source_scans.py outputs, so that fit_pointing_model.py could be used by both.